testing/tests/DevExpress.viz.gauges/barGauge.tests.js

```diff
@@ -574,6 +575,104 @@ QUnit.test('Some values are not changed', function(assert) {
     }, this);
 });
 
+QUnit.test('label location if the angle is < 90 and resolveLabelOverlapping - shift', function(assert) {
+    const done = assert.async();
+    this.$container.dxBarGauge({
+        values: [10, 11, 12],
+        resolveLabelOverlapping: 'shift'
+    });
+    const group = this.getBarsGroup();
+
+    group.animationComplete = $.proxy(function() {
+        assert.deepEqual(group.children[2]._stored_settings.points, [64.04786, 219.49845, 21, 7], 'line 1 coords');
```

Comment:
cannot we test it without a private api?
Then a suggest you to introduce a separate module for this tests and create a getter for this private api in `beforeEach` section to simplify possible refactoring if private api is changed

Reply:
I partially agree with this, but here let's create a refactoring task and fix the tests in it, where a private api is used